src/FileHelpers.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace dnj\Ticket;
+
+use dnj\Filesystem\Contracts\IFile;
+
+trait FileHelpers
+{
+    protected function saveFile(string $attachFilePath, string $extension, IFile $file): IFile
+    {
+
+        $md5Hash = @md5_file($attachFilePath);
```

Comment:
why ignoring errors? that's make a huge headache when someone want to debug your code.